css/login/login-body.css

```diff
@@ -0,0 +1,73 @@
+.body{
```

Comment:
I was relying on my display instead of using the inspect function which is why I decided to ignore the values on Figma. I've now changed it so that it is exactly like the mock-up.